src/main/java/edu/kis/powp/jobs2d/command/historyComponent/HistoryWindowCommandChangeObserver.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.command.historyComponent;
+
+import edu.kis.powp.observer.Subscriber;
+
+public class HistoryWindowCommandChangeObserver implements Subscriber {
+
+    private HistoryManagerWindow backLogManagerWindow;
+    public HistoryWindowCommandChangeObserver(HistoryManagerWindow commandManagerWindow) {
+        super();
+        this.backLogManagerWindow = commandManagerWindow;
+    }
+
+    @Override
+    public void update() {
+        backLogManagerWindow.updateCommandListField();
```

Comment:
Try to redesign the history component to be independent of the GUI. (e.g. we might want to change GUI or use history from another component without accessing GUI).
There should be a separate class representing the history (it makes more sense if history would be more than a text - returning to previous commands).